client/src/app/site/motions/models/view-motion-amended-paragraph.ts

```diff
@@ -111,4 +111,8 @@ export class ViewMotionAmendedParagraph implements ViewUnifiedChange {
     public showInFinalView(): boolean {
         return this.amendment.state && this.amendment.state.merge_amendment_into_final === MergeAmendment.YES;
     }
+
+    public isTitleChange(): boolean {
+        return false; // Not implemented for amendments
```

Comment:
If amendments are an open todo, highlight it like so:
```ts
/**
 * TODO: Not implemented for amendments
*/
```

Otherwise ignore

Reply:
Not an open ToDo.